js/script.js

```diff
@@ -0,0 +1,67 @@
+console.log('hello is this showing up?');
+//The user types in the name of the city
+//The user clicks on the submit button, which carries out the function findCity
+//The function is prevented from going into the default action
+//The function extracts and stores the user input
+//the user may use lowercase, so make sure strings are converted to lowercase.
+//The stored name of the city is then evaluated to a set of conditionals
+function error() {
+  alert('Oops! There is no picture for this city!');
+}
+
+function findCity(event) {
+  event.preventDefault(); 
+  console.log('submitting the form');
+
+  var city = $('#city-type').val();
+  city = city.toLowerCase();
+    
+  if (city === 'new york' || city === 'new york city'|| city === 'ny' || city === 'nyc') {
+    $('body').removeClass().addClass('nyc');
+  } else if (city === 'san francisco' || city === 'san fran' || city === 'bay area' || city === 'sf') {
+    $('body').removeClass().addClass('sf');           
+  } else if (city === 'los angeles' || city === 'la' || city === 'lax') {
+    $('body').removeClass().addClass('la');           
+  } else if (city === 'austin' || city === 'atx') {
+    $('body').removeClass().addClass('austin');
+  } else if (city === 'sydney' || city === 'syd') {
+    $('body').removeClass().addClass('sydney');      
+  } else {
+    $('body').removeClass();
```

Comment:
Since in every single case you're calling `$('body').removeClass()`, you can DRY (**d**on't **r**epeat **y**ourself) up your code by calling `$('body').removeClass();` before your entire `if` block.

e.g.

```js
// ...
city = city.toLowerCase();
$('body').removeClass();

if (city === 'new york' || city === 'new york city'|| city === 'ny' || city === 'nyc') {
  $('body').addClass('nyc');
} else if (/* ... */) {
  $('body').addClass('sf'); 
} // etc...
```
In this code structure, you wouldn't even need the `else` since you already handled `$('body').removeClass();` before the `if` block.

Reply:
If I use this structure , how can I still get the alert box without using else?

Reply:
Ah excellent point! So you would in fact need the `else` but now you would only need to call `return error();`